openreview/journal/process/submission_decision_pre_process.py

```diff
@@ -3,3 +3,10 @@ def process(client, edit, invitation):
     if edit.note.content.get('recommendation', {}).get('value') == 'Reject' and edit.note.content.get('certifications', {}).get('value', []):
         raise openreview.OpenReviewException(f'Decision Reject can not have certifications')
 
+    if edit.note.content.get('claims_and_evidence', {}).get('value') == 'Yes' and edit.note.content.get('audience', {}).get('value') == 'Yes':
```

Comment:
are these fields specific to TMLR?

I believe it doesn't affect other venues if these fields don't exist in the decision invitation.

Reply:
They are not specific to TMLR, they exist in the decision invitation for the journal but are textareas, so I thought this wouldn't affect them.